src/sweeper/__main__.py

```diff
@@ -41,15 +42,14 @@ def main():
         empty_sweeper.sweep()
 
         report_data = empty_sweeper.get_report()
-        report_string = helper.format_report(report_data, 'empties')
-
         if args['<report_path>']:
-            helper.save_report(args['<report_path>'])
+            report.save_report(report_data, 'empties', args['<table_name>'], args['<report_path>'])
+        else:
+            report.print_report(report_data, 'empties', args['<table_name>'])
 
         if args['--try-fix']:
             empty_sweeper.try_fix()
 
-         print(empty_sweeper.get_report())
```

Comment:
I think we will need a status report of how well the tool did when trying to fix the issues. Don't you think?